### PR TITLE
Add keybindings for shapes

### PR DESCRIPTION
**Pull request by vjeux**
I'm not 100% sure about this one. I feel like it's going to help people be a lot more productive to display the key bindings at all time. But it also clutters the UI...

I tried putting both the letter and number but it looked really terrible. I feel like using the numbers is going to be more predictable anyway.

ezgif-3-8f13f6b4b748

### CODE REVIEW

**Comment by dwelle** [comment]
I think the numbers on tools are fine, but if we really wanted to keep the UI clean, an alternative could be that the keybindings are shown only when you hold `Alt`.

**Comment by dwelle** [comment]
For me the numbers are super tiny, though:

image

**Comment by lipis** [comment]
I like it!! Do we really need the letter bindings? I think we are good with just numbers.

**Comment by dwelle** [comment]
@lipis  I think we should keep letters at least for well-known tools, such as selection (albeit remap it to `V` key). Not sure what other standards are there for the other tools.

Btw, are the numbers that tiny for you, too?

**Comment by lipis** [comment]
I think we could increase the size a bit.. yes

**Comment by Fausto95** [comment]
Right now the numbers key bindings aren't working on my azerty keyboard

**Comment by dwelle** [comment]
@Fausto95 I'll take a look.

I've increase font-size to 0.5em because anything below it didn't make any difference on my screen, not sure why. Lemme know if the size looks right for you guys.

**Comment by dwelle** [comment]
Should be fixed for non-qwerty keyboards, too.

**Comment by Fausto95** [comment]
It's working! thanks @dwelle !

File: src/components/ToolIcon.scss [hunk 1]
```diff
@@ -83,3 +83,12 @@
     box-shadow: none;
   }
 }
+
+.ToolIcon__keybinding {
+  position: absolute;
+  bottom: 2px;
+  right: 2px;
```
Comment (lipis) [inline src/components/ToolIcon.scss]:
```suggestion
  right: 4px;
```

I think it looks a bit better like this.. or maybe `3px`.. WDYT?

Comment (lipis) [reply src/components/ToolIcon.scss]: At the moment they look to tight to the border.

File: src/components/ToolIcon.scss [hunk 2]
```diff
@@ -83,3 +83,12 @@
...
+  font-size: 0.5em;
+  color: #aaa;
```
Comment (lipis) [inline src/components/ToolIcon.scss]: Can we use any of these? https://yeun.github.io/open-color/

File: src/components/ToolIcon.scss (hunk 1 above)
Comment (dwelle) [reply src/components/ToolIcon.scss]: used 3px

**Comment by dwelle** [comment]
Merging?

**Comment by Fausto95** [comment]
Let’s merge this!

**Comment by lipis** [comment]
Should we increase those, or ideally using the same style from them?

Screenshot 2020-02-01 at 7 41 07 PM

**Comment by dwelle** [comment]
Those are actually already larger (0.7 vs 0.5).

**Pull request merged**